src/containers/AppContainer.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+'use strict';
+
+import React, { Component } from 'react';
+import {
+  Dimensions,
+  ListView,
+  StyleSheet,
+  View
+} from 'react-native';
+import VFRChartsList from '../components/VFRChartsList';
+import Header from '../components/Header';
+import Menu from '../components/Menu';
+import Settings from '../components/Settings';
+import VFRChart from '../model/VFRChart';
+import Scenes from './Scenes';
+import SideMenu from './SideMenu';
+import Colors from '../styles/Colors';
+import { getSavedCharts } from '../utility/StorageUtility';
+
+const statusBarHeight = 0;
+const menuWidth = 200;
+const screenHeight = Dimensions.get('window').height;
+
+class AppContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      route: Scenes.HOME,
+      openMenu: false,
+      savedVfrCharts: [],
+    };
+  }
+
+  componentDidMount() {
+    var savedVfrCharts = getSavedCharts().map(function(chart) {
+      return new VFRChart(chart);
+    });
+
+    this.setState({
+      savedVfrCharts: savedVfrCharts,
+    });
+  }
+
+  handleHeaderPress() {
```

Comment:
I will be adding callback functions that are called when a chart is clicked in this class.